src/overlay/test/OverlayTestUtils.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+// Copyright 2022 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include <memory>
+
+namespace stellar
+{
+
+class Application;
+
+namespace overlaytest
```

Comment:
please use overlaytestutils to be consistent with the rest of the codebase

Reply:
Done!